src/target/espressif/esp32s2.c

```diff
@@ -697,7 +514,6 @@ struct target_type esp32s2_target = {
 
 	.checksum_memory = xtensa_checksum_memory,
 
-	.get_gdb_arch = xtensa_get_gdb_arch,
```

Comment:
We will use `target_get_gdb_arch()`  later on to differentiate from riscv.  Any reason to remove this?